internal/quirks/manufacturer/manufacturer.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manufacturer
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openconfig/ondatra"
+)
+
+// PortChannelName returns the port channel interface name which is
+// vendor specific.
+func PortChannelName(t testing.TB, dut *ondatra.DUTDevice, i int) string {
+	t.Helper()
+	if i <= 0 {
+		t.Fatalf("port-channel index must be >= 1: %d", i)
```

Comment:
Is this true across all implementations? I think we can have `ae0` in JUNOS. I'd recommend not having validations in this function.

Reply:
The original internal version had a comment addressing this very question, but somehow the comment got lost. Nevertheless, we're not using this function anymore.